features/importing_service_definitions.feature

```diff
@@ -10,7 +10,7 @@ Feature: Importing service definitions
       extensions:
         Zalas\Behat\NoExtension:
           imports:
-            - features/bootstrap/config/services.yml
+            - %paths.base%/features/bootstrap/config/services.yml
```

Comment:
Giving it a bit more thought, I'd just remove it from the scenario (and not add a new one). I think adding the path is redundant. We can support it just in case someone uses it (as they expect it to work in behat.yml), but there's no reason to encourage it in documentation.